### ISSUE TITLE
`torch.clamp`  documentation refers `value` as an argument. 

### ISSUE THREAD

**Issue by DNXie** [pytorch/pytorch#43497]
#### Link

https://pytorch.org/docs/stable/generated/torch.clamp.html#torch.clamp

#### API

`torch.clamp`

#### Issue

In the "Parameters" section under the following two signature:

- `torch.clamp(input, *, min, out=None) → Tensor`
- `torch.clamp(input, *, max, out=None) → Tensor`

there is an input argument `value` , but it is not in the signature, and it is not accepted by the function. It should be `min ` and `max` respectively

![image](https://user-images.githubusercontent.com/24580222/91067305-bdfbb680-e600-11ea-8124-9ca4613ae37c.png)

![image](https://user-images.githubusercontent.com/24580222/91067329-c5bb5b00-e600-11ea-98ab-5cbb6af8d4a2.png)

Running code :

~~~python
import torch
a = torch.randn(4)
torch.clamp(a, max=0.5, value =1)
~~~

gives exception 

~~~python
TypeError: clamp() got an unexpected keyword argument 'value'
~~~

#### **System information**

- OS: MacOS Mojave 10.14
- PyTorch version: 1.7.0.dev20200819+cpu
- Python version: 3.8.2

cc @jlin27

**Comment by ngimel** [comment]
Thank you for reporting, we'll accept a PR fixing this.

**Comment by simaki** [comment]
I think this issue has been fixed by #43583.

**Comment by ngimel** [comment]
Fixed on master, closing.
